docs/platforms/javascript/common/apis.mdx

```diff
@@ -0,0 +1,942 @@
+  signature="function init(options: InitOptions): Client | undefined"
+>
+  Initialize the SDK with the given options. See{" "}
+  <PlatformLink to="/configuration/options/">Options</PlatformLink> for the
+  options you can pass to `init`.
+</SdkApi>
+
+<SdkApi name="getClient" signature="function getClient(): Client | undefined">
+  Returns the currently active client.
+</SdkApi>
+
+<SdkApi
+  name="setCurrentClient"
+  signature="function setCurrentClient(client: Client): void"
+>
+  Make the given client the current client. You do not need this if you use
+  `init()`, this is only necessary if you are manually setting up a client.˝
+</SdkApi>
+
+<SdkApi
+  name="lastEventId"
+  signature="function lastEventId(): string | undefined"
+>
+  Returns the ID of the last sent error event. Note that this does not guarantee
+  that this event ID exists, as it may have been dropped along the way.
+</SdkApi>
+
+<SdkApi
+  name="flush"
+  signature="function flush(timeout?: number): Promise<boolean>"
+  parameters={[
+    {
+      name: "timeout",
+      type: "number",
+      description:
+        "Maximum time in ms the client should wait to flush its event queue. Omitting this parameter will cause the client to wait until all events are sent before resolving the promise.",
+    },
+  ]}
+>
+  Flushes all pending events.
+</SdkApi>
+
+<SdkApi name="isEnabled" signature="function isEnabled(): boolean">
+  Returns true if the SDK is initialized & enabled.
+</SdkApi>
+
+<SdkApi
+  name="close"
+  signature="function flush(timeout?: number): Promise<boolean>"
+  parameters={[
+    {
+      name: "timeout",
+      type: "number",
+      description:
+        "Maximum time in ms the client should wait to flush its event queue. Omitting this parameter will cause the client to wait until all events are sent before resolving the promise.",
+    },
+  ]}
+>
+  Flushes all pending events and disables the SDK. Note that this does not
+  remove any listeners the SDK may have set up.
+</SdkApi>
+
+<SdkApi
+  name="addEventProcessor"
+  signature="function addEventProcessor(processor: EventProcessor): void"
+  parameters={[
+    {
+      name: "processor",
+      type: "(event: Event, hint: EventHint) => Event | null | Promise<Event | null>",
+    },
+  ]}
+>
+  Adds an event processor to the SDK. An event processor receives every event
+  before it is sent to Sentry. It can either mutate the event (and return it) or
+  return `null` to discard the event. Event processors can also return a
+  promise, but it is recommended to use this only when necessary as it slows
+  down event processing.
+</SdkApi>
+
+<SdkApi
+  name="addIntegration"
+  signature="function addIntegration(integration: Integration): void"
+>
+  Adds an integration to the SDK. This can be used to conditionally add
+  integrations after `Sentry.init()` has been called. Note that it is
+  recommended to pass integrations to `init` instead of calling this method,
+  where possible.
+</SdkApi>
```

Comment:
same q here - should we link to the [Integrations](https://docs.sentry.io/platforms/javascript/configuration/integrations/) page?